.github/workflows/smoke.yml

```diff
@@ -4,7 +4,7 @@ name: Smoke
 on:  # yamllint disable-line rule:truthy
   workflow_dispatch:
   pull_request:
-    branches: ["main"]
+    branches: ["revert-10647-revert-10558-amazimbe/use-new-maven-version"]
```

Comment:
You don't need to do this, it should stay `main`. 

This means when a PR is opened against `main` it will trigger the workflow. Putting in the name of the branch doesn't make sense, and will prevent the smoke test from running. 

It's ok if it fails for now given that the fix is on another branch in Core.

Reply:
I was advised to do this a few weeks ago and I obliged as I thought I would be able to merge my dependabot core PR if the smoke tests didn't go green